docs/src/pages/docs/functions/helpers.md

```diff
@@ -756,33 +867,46 @@ pipeS :: Semigroupoid s => (s a b, ..., s y z) -> s a z
 ```
 
 While `Star`s and `Arrow`s come in very handy at times, the only thing that
-could make them better is to compose them . With `pipeS` you can do just that
+could make them better is to compose them. With `pipeS` you can do just that
 with any `Semigroupoid`. Just like with [`composeS`](#composes), you just pass
 it `Semigroupoid`s of the same type and you will get back another `Semigroupoid`
 with them all composed together. The only difference between the two, is that
 `pipeS` composes in a left-to-right fashion, while [`composeS`](#composes) does
 the opposite.
 
 ```javascript
-import {
-  curry, isNumber, pipeS, prop, safeLift, Star
-} from 'crocks'
+import pipeS from 'crocks/helpers/pipeS'
+
+import curry from 'crocks/core/curry'
+import isNumber from 'crocks/predicates/isNumber'
+import prop from 'crocks/Maybe/prop'
+import safeLift from 'crocks/Maybe/safeLift'
+import Star from 'crocks/Star'
+
+// MaybeStar :: (a -> b) -> Star (Maybe (a -> b))
```

Comment:
So i took a crack at all the sig's for `Star` but i think i likely got them wrong because they don't look neat. Sorry for the extra steps but when you see this let me know the correct ones